src/mapd-crossfilter.js

```diff
@@ -82,6 +82,22 @@ function notEmpty(item) {
   }
 }
 
+/**
+ * creates WKT POLYGON string from given points array
+ * @param points, ex: [[-180,-90], [180.-90], [180,90], [-180,90]]
+ * @returns {string}
+ */
+export function createWKTPolygonFromPoints(points) {
```

Comment:
Is an empty array `[]` a valid points array? This function would fail if so.

Reply:
@jrajav , thank you for catching it up. I added validation. I was too excited.